packages/azure-kusto-data/src/connectionBuilder.ts

```diff
@@ -45,17 +46,24 @@ export class KustoConnectionStringBuilder extends KustoConnectionStringBuilderBa
     static withAadApplicationCertificateAuthentication(
         connectionString: string,
         aadAppId: string,
-        applicationCertificatePrivateKey: string,
+        applicationCertificatePrivateKeyOrPath: string,
```

Comment:
I really don't like this approach, why do we have to guess what they meant?
Make it into an optional param, and add another one